app/views/dashboard/_past_applications.haml

```diff
@@ -5,7 +5,7 @@
       .note_item
         = select_tag :department_ids,
           options_from_collection_for_select(@departments, :id, :name),
-        class: 'dept_select', multiple: true
```

Comment:
@werebus everything is working now. Thank you for your patience in reteaching me things like 3 times before I get them maybe.